apps/OboeTester/app/src/main/cpp/analyzer/BaseSineAnalyzer.h

```diff
@@ -195,7 +213,9 @@ class BaseSineAnalyzer : public LoopbackProcessor {
     double  mOutputAmplitude = 0.75;
     // If this jumps around then we are probably just hearing noise.
     double  mPhaseOffset = 0.0;
+    static constexpr double kPhaseInvalid = -999.0;
     double  mMagnitude = 0.0;
+    static constexpr double kMinValidMagnitude = 2.0 / (1 << 16);
```

Comment:
Why do we need both kMinValidMagnitude and kPhaseInvalid?

Reply:
I added a comment.